GitUI/CommandsDialogs/FormDeleteBranch.cs

```diff
@@ -19,9 +21,13 @@ public sealed partial class FormDeleteBranch : GitModuleForm
             new("You cannot delete unmerged branch until you set “force delete” mode.");
         private readonly TranslationString _cannotDeleteCurrentBranchMessage =
             new("Cannot delete the branch “{0}” which you are currently on.");
+        private readonly TranslationString _cannotDeleteWorktreeBranchMessage =
+            new("Cannot delete branches:" + Environment.NewLine + "{0}");
+        private readonly TranslationString _branchPathMessage = new(" - “{0}” checked out at “{1}”");
```

Comment:
❗ We must not use extended charset
```suggestion
        private readonly TranslationString _branchPathMessage = new(" - \"{0}\" checked out at \"{1}\"");
```